packages/nextjs/test/config/loaders.test.ts

```diff
@@ -98,14 +72,14 @@ describe('webpack loaders', () => {
         test: /sentry\.client\.config\.(jsx?|tsx?)/,
         use: [
           {
-            loader: expect.stringEndingWith('prefixLoader.js'),
-            options: expect.objectContaining({ templatePrefix: 'clientRewriteFrames' }),
+            loader: expect.stringEndingWith('valueInjectionLoader.js'),
+            options: expect.objectContaining({ values: expect.objectContaining({}) }),
           },
         ],
       });
     });
 
-    it('adds release prefix loader to client config', async () => {
+    it('adds `valueInjection` loader with release values to client and server configs', async () => {
```

Comment:
H: This only tests the client side, since `incomingWebpackConfig` is `clientWebpackConfig`.

Options:
- Add these values (and the other ones we inject) into the above two tests.
- Take this out of the client-specific `describe` and make it an `it.each` with test cases for server and client.

Reply:
I guess it would have tested the serverside too since both of the loaders are added to the config, no matter if we're on the client or on the server. The provided config doesn't really matter here because whether the loader is injected or not doesn't depend on the config.

Anyhow, the suggested change in regards to adding the isomorphic values to client and server values made this test redundant, so I removed it.